core/utilities/user_resource_map.py

```diff
@@ -1,39 +1,36 @@
 # Created by Kayla Patterson 07/16/2020
 # This file maps each user to their perspective
-# resources and exports it as a pickle file 
```

Comment:
Do you use pylint or something? It seems like there is some auto formating going on here

Reply:
I think I did that haha, I always put a space between the hashtag and comment i dont know why